src/Pages/Overview/App.js

```diff
@@ -0,0 +1,343 @@
+/*
+NOTE!
+
+There are 2 search API in user:
+"search" get an array of all users, need a token
+"searchFor" get an {obj} of the user, need a token and an emailOptIn
+
+Log out "this.state.users" you will see an array of users
+
+Direction!!!!!! its gonna be pretty tricky:
+1.Delete what you dont use/need
+2.use SVG for icons
+3.use map() put individual user in "this.state.users" under the board
+      ex: this.state.users.map(user => {html format})
+4.Sort by "members, officers, excutives, alumni,...dont remmeber"
+      look into api>models>User
+      also give numbers users in each group
+5.search bar, shouldnt be too hard, just add filter to the users array
+6.There should be a delete route in api>routes>user
+      delete api should need a token and an email (to be deleted)
+
+Ya are my ace good luck.
+*/
+
+import React, { Component } from 'react'
+import './App.css'
+import InfoCard from '../Profile/admin/AdminView'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+
+import axios from 'axios'
+class Login extends Component {
```

Comment:
Take a look at how we used hooks in the other file and use them here. Also this component should't be named login.